straxen/plugins/peaklet_processing.py

```diff
@@ -207,6 +225,141 @@ def clip_peaklet_times(peaklets, start, end):
                 p['length'] = (end - p['time']) // p['dt']
 
 
+@numba.jit(nopython=True, nogil=True, cache=True)
+def peak_saturation_correction(records, peaks, to_pe,
+                               reference_length=100,
+                               min_reference_length=20,
+                               use_classification=False,
+                               ):
+    """Correct the area and per pmt area of peaks from saturation
+    :param records: Records
+    :param peaks: Peaklets / Peaks
+    :param reference_length: Maximum number of reference sample used
+    to correct saturated samples
+    :param min_reference_length: Minimum number of reference sample used
+    to correct saturated samples
+    :param use_classification: Choice to use peak type
+    """
+
+    if not len(records):
+        return
```

Comment:
is this None the right dtype?

Reply:
Now it should be in https://github.com/XENONnT/straxen/pull/244/commits/2198316c7b312165cd2e21a584d14e950a0ed23d. With the last return removed